manifests/dev/virt-api.yaml.in

```diff
@@ -0,0 +1,50 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: virt-api
+  namespace: kube-system
```

Comment:
use {{ namespace }} instead.

Reply:
ugh, copy paste error, thanks for catching that.